config/routes.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   namespace :users do
-    resources :applications do
+    resources :service_providers do
     end
   end
 
```

Comment:
I don't think we need to address this any time soon, but note that service provider entries will need to have delegated access at some point.